zh/3d/terrain.md

```diff
@@ -65,31 +51,26 @@ Terrain::DetailMap b("road.dds");
 Terrain::DetailMap a("greenSkin.jpg");
 {%- endcodetabs %}
 
-* to create a `TerrainData` variable with __detail maps__, you need to specify
-the terrain's __height map__ file path and __alpha map__ file path:
+* 构造 `Terrain::TerrainData` 对象, 构造函数的第一个参数为高度图的路径，第二个参数为 Alpha 贴图的路径，接下来的四个参数为之前所构造的四个 DetailMap 结构体.
 
 {% codetabs name="C++", type="cpp" -%}
 Terrain::TerrainData data("chapter9/heightmap16.jpg","TerrainTest/alphamap.png", r, g, b, a);
 {%- endcodetabs %}
 
-* pass the `TerrainData` object to __Terrain::create__, the last parameter determines
-the LOD policy (as talked about above). Example:
+* 调用 `Terrain::create` 创建出地形, 第一个参数是之前构造好的 `TerrainData` 对象, 第二个参数是细节级别策略
 
 {% codetabs name="C++", type="cpp" -%}
 _terrain = Terrain::create(data, Terrain::CrackFixedType::SKIRT);
 {%- endcodetabs %}
 
-* If you set a `Terrain` objects __camera mask__ and add it to a `Node` or a
-`Scene`, be careful. When `Terrain` is added into a `Node` or a `Scene`, you can
-not use __transform(translate, scale)__ on it anymore. If you do this after calling
-__addChild()__, some of the terrain's methods may calculate wrong results.
+* 请注意, 在地形对象被添加到节点或场景中(进行了 `addChild()` 操作)后, 就不能再对地形对象使用转换, 比如缩放大小. 否则, `Terrain` 的方法可能会造成难以预料的结果.
+
+### 获取地形高度
 
-### Get Terrain Height
-Use the method __Terrain::getHeight(float x, float z, Vec3 * normal= nullptr)__ to
-get the specified position's height. This method is very useful when you want to
-put a `Sprite3D` object or any `Node` on the terrain's surface.
+使用方法 `Terrain::getHeight(float x, float z, Vec3 * normal= nullptr)` 可以获取到地形的高度, 当你想把一个节点对象放到地形表面的时候, 这个方法会非常有用.
 
 ### Ray-Terrain intersection test
```

Comment:
I can't understand this paragraph means, title is `Ray-Terrain intersection test`, but most of contents talk about  LOD policy
```
A __Ray-Terrain__ intersection test will calculate the intersection point by giving
  a specified position.

__Terrain::CrackFixedType::SKIRT__ will generate four skirt-like meshes at each
		 
__Terrain::CrackFixedType::SKIRT__ will generate four skirt-like meshes at each
```
is it unfinished? @slackmoehrle

Reply:
This text came from an employee that no longer works here. We may need to ask the current 3D developer to take a look at this. I am not sure who is the current 3D dev, do you know?

Reply:
I don't know either. I will beg @minggo review, after this chapter finished

Reply:
I think I recall the previous developer being named Tony